probe/http/http.go

```diff
@@ -133,6 +140,17 @@ func (h *HTTP) DoProbe() (bool, string) {
 		if resp.StatusCode >= 500 {
 			message = fmt.Sprintf("HTTP Status Code is %d", resp.StatusCode)
 			status = false
+		} else if len(h.SuccessCode) > 0 {
+			var hitted bool
```

Comment:
the `hitted` wrong word, it should be `hit`. But I think the `valid` would make sense.